### ISSUE TITLE
Group by multiple columns

### ISSUE THREAD

**Issue by AndreiKingsley** [JetBrains/lets-plot-kotlin#136]
Is it possible to make a group by several columns (without mappings to other aesthetics)? For example like this: 
`group = listOf("col1", "col2")`

**Comment by alshan** [comment]
No, collection of column names is not supported as a value of aesthetic. It must be either a name or a collection of values (data).

**Comment by alshan** [comment]
Just stumbled upon the "interaction" fun in R: 
`group = interaction(dir, grp)`

https://stackoverflow.com/questions/50604055/ggplot2-geom-bar-position-failure

**Comment by alshan** [comment]
In [v4.12.0](https://github.com/JetBrains/lets-plot-kotlin/releases/tag/v4.12.0) `group` can accept a list of column names.

There were also other changes in relation to `group` aesthetic which are potentially very BREAKING for Kandy. Please read the release notes carefully.
